assets/src/edit-story/components/library/panes/media/media3p/providerTab.js

```diff
@@ -92,17 +67,17 @@ function ProviderTab({
   );
 
   return (
-    <Tab
+    <Pill
```

Comment:
https://github.com/google/web-stories-wp/issues/6874

Reply:
@samwhale want me to just add these fixes to this PR? or just revert the pill changes here if you've already started work on it?

Reply:
ope. seeing it here. nvmd
https://github.com/google/web-stories-wp/pull/6977#issuecomment-811968172

Reply:
Oh yeah, just do it here! I removed my pill changes since I'm not using the pills